README.md

```diff
@@ -78,145 +135,201 @@ See [third_party_integration](./third_party_integration) for examples of code mo
 ## System requirements
 - Ubuntu\* 18.04 or later (64-bit)
 - Python\* 3.6.2 or later
-- NVidia CUDA\* Toolkit 10.2 or later^
-- PyTorch\* 1.5 or later (1.8.0 not supported, 1.8.1 supported)
+- Supported frameworks:
+  - PyTorch\* 1.5 or later (1.8.0 not supported, 1.8.1 supported)
```

Comment:
@ljaljushkin Thanks for your comment. There is torch>=1.5.0, <=1.8.1, !=1.8.0" condition in the `setup.py`. @vshampor, what version Pytorch we should specify in the Readme? `PyTorch\* >=1.5.0, <=1.8.1 (1.8.0 not supported)` or `PyTorch\* 1.8.1`?

Reply:
1.8.1 is BKC, but we "should" support the others in setup.py as wel, so I think the former variant should be specified.

Reply:
IMHO, user will be confused with the former variant: `PyTorch\* 1.5 or later (1.8.0 not supported, 1.8.1 supported)`

PyTorch 1.5 or **later** is supported, but 1.8.1 is somehow **more supported**? What's the difference?
Does it mean that 1.7 and 1.9 equally supported?

1.9 is not working for sure. We shouldn't recommend to install it. 
And we can't be 100% sure that 1.7 is also supported, because we don't test it. 

I wouldn't mislead users with unconfirmed claims. 
We can explicitly say that we tested NNCF for PT with 1.8.1. You could try other versions, but we don't guarantee operability.

Reply:
@vshampor, @ljaljushkin Thanks for your comments. I have updated to `PyTorch\* >=1.5.0, <=1.8.1 (1.8.0 not supported)`. I hope the following note clears up any misunderstandings: 
```
 This repository is tested on Python* 3.6.2+, PyTorch* 1.8.1 (NVidia CUDA\* Toolkit 10.2) and TensorFlow* 2.4.2 (NVidia CUDA\* Toolkit 11.0).
```